src/components/SafetyScore/SafetyScore.tsx

```diff
@@ -1,20 +1,20 @@
 import { getProgress } from './variants';
 import SafetyBadge from './SafetyBadge';
 
-type SafetyScorePropsType = Readonly<{
+type SafetyScoreType = Readonly<{
   danger: number;
 }>;
 
-export default function Score({ danger }: SafetyScorePropsType) {
-  const { label, value, color } = getProgress(danger);
+export default function Score({ danger }: SafetyScoreType) {
+  const { label, value, bg } = getProgress(danger);
 
   return (
     <div className="flex items-center">
       <SafetyBadge danger={danger} large />
       <div className="ml-4 flex-1">
         <div className="text-sm font-semibold"> {label}</div>
         <div className="h-2 w-full bg-gray bg-opacity-20 rounded-full mt-1">
-          <div className={`w-${value} h-2 rounded-full bg-${color}`}></div>
```

Comment:
the dynamic classes worked fine locally, but not on prod

Reply:
Unfortunately you cannot build class names like that. Purge script looks for a full names.
https://tailwindcss.com/docs/optimizing-for-production

please take a look at do’s and don’ts

Reply:
yeah, that's why I changed it

Reply:
works now